controller_manager:
  ros__parameters:

    skill_controller:
      type: rackki_learning/SkillController

skill_controller:
  ros__parameters:
    model_path: /home/scherzin/src/robot_folders/checkout/rack-ki/colcon_ws/src/rackki_learning/test/models/model_1
